changelog_generator/__main__.py

```diff
@@ -53,10 +55,10 @@ def get_commit_but_types(
 
 
 def main() -> None:
-    repository = RepositoryManager("./")
+    repository = RepositoryManager("./", os.environ.get("SERVICE"))
```

Comment:
BTW this cannot be a env variable, it should be an arguments give the binary.

Reply:
I'm pretty sure that that github action allows environment variables to be set. something like 
```yaml
....
    env:
      - MY_VAR: my value
```

Reply:
Indeed, we can use it this way.

Reply:
BTW you have to update the readme :)